src/AppBar/AppBar.js

```diff
@@ -27,23 +28,29 @@ export const styleSheet = createStyleSheet('AppBar', (theme) => {
   };
 });
 
-export default class AppBar extends Component {
-  static propTypes = {
-    accent: PropTypes.bool,
-    children: PropTypes.node,
-    className: PropTypes.string,
-    primary: PropTypes.bool,
-  };
+type DefaultProps = {
+  primary: boolean,
+};
 
-  static defaultProps = {
-    primary: true,
-  };
+type Props = {
+  accent?: boolean,
+  children?: Object|string,
+  className?: string,
+  primary?: boolean,
+};
 
+export default class AppBar extends Component<DefaultProps, Props, void> {
```

Comment:
Yeah, that's true and when you define type of Component in such way you don't need to pass it [ secondary ](https://github.com/callemall/material-ui/pull/4630/files#diff-92e738aaae5e0f01790538d21dff9586R51), because it had already [passed to props](https://github.com/facebook/flow/blob/v0.27.0/lib/react.js#L20)